src/pages/iou/request/step/IOURequestStepDestination.tsx

```diff
@@ -112,56 +115,62 @@ function IOURequestStepDestination({
     };
 
     return (
-        <StepScreenWrapper
-            headerTitle={backTo ? translate('common.destination') : tabTitles[iouType]}
-            onBackButtonPress={navigateBack}
-            shouldShowWrapper={!openedFromStartPage}
-            shouldShowNotFoundPage={shouldShowNotFoundPage}
-            testID={IOURequestStepDestination.displayName}
+        <ScreenWrapper
+            includePaddingTop={false}
+            keyboardVerticalOffset={variables.contentHeaderHeight + top + variables.tabSelectorButtonHeight + variables.tabSelectorButtonPadding}
+            testID={`${IOURequestStepDestination.displayName} Wrap`}
```

Comment:
In case we'll keep this solution, let's rename this to follow `-` existing dash pattern.

```suggestion
            testID={`${IOURequestStepDestination.displayName}-container`}
```

Reply:
cc @Eskalifer1